components/validator/guardian.go

```diff
@@ -151,12 +430,46 @@ func (g *Guardian) tryConfirmTransaction(event *bindings.SecurityCouncilValidati
 	return nil
 }
 
-func (g *Guardian) checkConfirmCondition(transactionId *big.Int, l2BlockNumber *big.Int) (bool, error) {
-	outputIndex, err := g.getL2OutputIndexAfter(l2BlockNumber)
+func (g *Guardian) tryConfirmRequestDeletionTx(event *bindings.SecurityCouncilDeletionRequested) error {
+	needConfirm, err := g.checkConfirmCondition(event.TransactionId, event.OutputIndex)
 	if err != nil {
-		return true, fmt.Errorf("failed to get output index after. (l2BlockNumber: %d): %w", l2BlockNumber.Int64(), err)
+		return fmt.Errorf("failed to check confirm condition. (transactionId: %d): %w", event.TransactionId.Int64(), err)
 	}
 
+	if !needConfirm {
+		return nil
+	}
+
+	cCtx, cCancel := context.WithTimeout(g.ctx, g.cfg.NetworkTimeout)
+	defer cCancel()
+	output, err := g.l2ooContract.GetL2Output(utils.NewSimpleCallOpts(cCtx), event.OutputIndex)
+	if err != nil {
+		return fmt.Errorf("failed to get output from L2OutputOracle contract(outputIndex: %d): %w", event.OutputIndex.Uint64(), err)
+	}
+
+	isValid, err := g.ValidateL2Output(g.ctx, output.OutputRoot, output.L2BlockNumber.Uint64())
+	if err != nil {
+		return err
+	}
+
+	if isValid {
+		g.log.Info("output deletion is requested, but it's valid")
```

Comment:
In general, the log level `Warn` is used for the case of potential issues that may lead to errors or unexpected behavior in the future if not addressed. I think this is not the case, it's normal. Instead, I'll add log of transaction ID and output index.

Reply:
I thought that if one of the security council members requested an output deletion but it wasn't a valid request, guardian should be aware of it and check for possible vulnerabilities. As you said, I agree that the log level being a WARN is not suitable, but it would be good to have guardian be aware of it in some other way such as discord notification, ...

Reply:
Right. We would have another channel for this. 👍